public/app/features/plugins/plugin_loader.ts

```diff
@@ -222,6 +227,10 @@ interface PanelCache {
 const panelCache: PanelCache = {};
 
 export function importPanelPlugin(id: string): Promise<grafanaData.PanelPlugin> {
+  if (DEPRECATED_PANELS[id]) {
+    id = DEPRECATED_PANELS[id];
+  }
```

Comment:
Is this all we need? won't we need to change the panel type property in the dashboard migrator

Reply:
Proobably, thought it's handled by the panel migrator (setMigrationHandler), but it's not

Reply:
@torkelo yup -> you are right:
![image](https://user-images.githubusercontent.com/2376619/110806854-bf31bf80-8282-11eb-9b8c-8f0b01be8543.png)

Reply:
definitely will require more changes!